lib/routes/collection.js

```diff
@@ -572,25 +572,29 @@ var routes = function (config) {
       return res.status(400).send('Bad file');
     }
 
-    keys.forEach((key) => {
+    let docs = [];
+
+    for(const key of keys){
       const fileContent = req.files[key].data.toString('utf8');
-      const str = `[${fileContent.replace(/}\n(\s)?\{/gm, '},{').replace(/\{"\$oid":"(.+?)"}/gm, '"$1"')}]`;
+      const str = `[${fileContent.replace(/}\s*\n\s*\{/gm, '},{').replace(/\{"\$oid":"(.+?)"}/gm, '"$1"')}]`;
```

Comment:
Because I ran into '\r\n' in my file. The input file could be generated by other scripts and even manually, so there may be several new lines between records. That's why I added blanks both before and after.